contrib/babelfishpg_tsql/antlr/TSqlParser.g4

```diff
@@ -5136,7 +5178,7 @@ collation
     ;
     
 full_column_name
-    : (((server=id? DOT)? schema=id? DOT)? tablename=id? DOT)? column_name=id
+    : (((server=id? DOT)? schema=id? DOT)? tablename=id? DOT)? ( geospatial_col | column_name=id )
```

Comment:
Again the logic is correct but we are using `tablename` as a column_name here for geospatial?

Reply:
Done. Made a seperate Rule